exs/mundo_1/java/001.java

```diff
@@ -0,0 +1,12 @@
+/*
+Desafio 001
+    Problema: Crie um programa que escreva "Olá mundo" na tela.
+
+Resolução do problema:
+*/
+
+class Main{
+    public static void main(String[] args){
+        System.out.println("Hello, world"); // Printa "Hello, world"
```

Comment:
Remova esse comentário, não há necessidade. O problema em si já descreve e a linha em questão é auto-descritiva.

```suggestion
        System.out.println("Hello, world");
```